### DISCUSSION TITLE
Épinglage fichier sur les application dans la barre de tâche / Pinning files to applications in the taskbar

### DISCUSSION THREAD

**Discussion by Nano33320** [valinet/ExplorerPatcher#3850] [Q&A]
Bonjour
C'est mon premier post sur Github, après quelques post "pas au bon endroit" j'espère cette fois être au bon endroit pour ma demande...
J'utilise explorer patcher pour plusieurs raisons liées au manque/perte fonctionnalité de Windows 11
mais la plus importante pour mon usage concerne "l'épinglage" de fichiers sur le menu d'une application
![Capture d'écran 2024-10-28 083535](https://github.com/user-attachments/assets/60f0ed5c-771f-4dda-b49a-5e66fcb6babf)
J'aimerai retrouver cette fonctionnalité qui existait dans Win 10 et plus dans Win 11.
Explorer Patcher permet de la retrouver.
je suppose qu'une ou plusieurs ligne dans la base de registre en sont la cause.
Est-il possible d'avoir ce correctif/modification ?
Merci
---- Google translation ----
Hello
This is my first post on Github, after a few "not in the right place" posts I hope this time to be in the right place for my request...
I use explorer patcher for several reasons related to the lack/loss of functionality of Windows 11
but the most important for my use concerns "pinning" files on the menu of an application
![Screenshot 2024-10-28 083535](https://github.com/user-attachments/assets/60f0ed5c-771f-4dda-b49a-5e66fcb6babf)
I would like to find this functionality that existed in Win 10 and more in Win 11.
Explorer Patcher allows to find it.
I suppose that one or more lines in the registry are the cause.
Is it possible to have this fix/modification?
Thanks

**Comment by pyrates999** [comment]
did you try right clicking on one and selecting pin to list?

**Comment by Nano33320** [comment]
Oui, impossible d'épingler un fichier sous Windows 11 je suis sous 23H2,  quid de 24H2 à venir ?
-------------------------------
Yes, impossible to pin a file under Windows 11 I am under 23H2, what about 24H2 to come?

**Comment by pyrates999** [comment]
do you not see `pin to list` when you right click on one?

**Comment by Nano33320** [comment]
voilà ce que j'ai avec le click droit 
![image](https://github.com/user-attachments/assets/16b439f2-a900-481c-a90c-4befc2fe7549)

**Reply by pyrates999** [reply]
Try enabling the windows 11 context menu: https://github.com/valinet/ExplorerPatcher/wiki/All-features#disable-the-windows-11-context-menu

**Comment by Nano33320** [comment]
Merci beaucoup pour ce lien vers un mode d'emploi complet d'Explorer Patcher. Ne parlant pas anglais, j'ai eu un peu de soucis (tests successifs) pour paramétrer à peu près l'outil.

Toutefois, je pense avoir mal exprimé mes attentes dans mon post.

J'ai utilisé Explorer Patcher jusqu'au "non fonctionnement" du click droit pour l'accès au menu "démarer". J'ai dû le supprimer en attendant les correctifs à ce sujet et la validation pour le passage à Win 24H2. Je le réinstallerai certainement dès que possible.
problème =>
![image](https://github.com/user-attachments/assets/36422e46-e90d-448e-a8d5-1f4495731208)

Ma question concerne les lignes de l'éditeur de registre qui ont été modifiées dans Win 10 vs Win 11 concernant "l'épinglage" des fichiers sous les applications dans la barre de tâche.
![Capture d'écran 2024-10-28 083535](https://github.com/user-attachments/assets/9ae6f4a1-0dfe-4183-bfed-edaaeaa50893)
J'aimerais retrouver cette fonctionnalité dans Windows 11 même sans "Explorer Patcher".
Merci de votre aide
--------------------------------------------------------------------
Thank you very much for this link to a complete user guide for Explorer Patcher. Not speaking English, I had a bit of trouble (successive tests) to configure the tool more or less.

However, I think I expressed my expectations badly in my post.

I used Explorer Patcher until the right click "did not work" to access the "start" menu. I had to delete it while waiting for fixes on this subject and validation for the transition to Win 24H2. I will certainly reinstall it as soon as possible.
 problem => 
![image](https://github.com/user-attachments/assets/36422e46-e90d-448e-a8d5-1f4495731208)

My question concerns the lines of the registry editor that have been modified in Win 10 vs Win 11 concerning the "pinning" of files under the applications in the taskbar.
![Capture d'écran 2024-10-28 083535](https://github.com/user-attachments/assets/56cdeb8d-b7c0-4e75-ae34-93821f033676)
I would like to find this functionality in Windows 11 even without "Explorer Patcher".
Thank you for your help

**Reply by pyrates999** [reply]
Please wait for a developer to respond.

**Reply by Nano33320** [reply]
Ok, merci, j'attendrais
----------------------
Ok, thanks, I'll wait
